module-services/service-evtmgr/battery/BatteryState.cpp

```diff
@@ -1,7 +1,8 @@
-// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
 #include "BatteryState.hpp"
+#include <evtmgr/battery/Thresholds.hpp>
```

Comment:
How about injecting these parameters via BatteryState constructor args instead of include which, by the way,  comes from product-specific targets? This way, we would have a nice generic module implementing simple battery management :)

Reply:
done